monitor/xmonitor/monitor.go

```diff
@@ -21,18 +20,6 @@ func Start(
 	cprovider cchain.Provider,
 	rpcClients map[uint64]ethclient.Client,
 ) error {
-	{
```

Comment:
nit: can remove this change as this only relates to monitor which isn't applicable to releases